portfolio/src/main/webapp/countries_game.js

```diff
@@ -0,0 +1,1137 @@
+//based from https://developers.google.com/public-data/docs/canonical/countries_csv; edited certain countries
```

Comment:
maybe put the countries data in a separate .js file since it's long, and it'll make the separation between data & logic clearer